### PR TITLE
Added/Update new Enum and View required for Fortify Integration

### PR DESCRIPTION
**Pull request by msenmurugan**
Added/Update new Enum and View required for Fortify Integration

### CODE REVIEW

File: src/main/java/com/blackducksoftware/integration/hub/model/view/components/MetaView.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+public final class MetaView extends HubComponent {
+    private final List<AllowEnum> allow;
+
+    private final String href;
+
+    private final List<LinkView> links;
+
+    public MetaView(List<AllowEnum> allow, String href, List<LinkView> links) {
+        super();
```
Comment (jamesrichard91) [inline src/main/java/com/blackducksoftware/integration/hub/model/view/components/MetaView.java]: Please delete the super(); from the constructor

File: src/main/java/com/blackducksoftware/integration/hub/model/view/components/MetaView.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        this.allow = allow;
+        this.href = href;
+        this.links = links;
+    }
+
+    public class LinkView extends HubComponent {
```
Comment (jamesrichard91) [inline src/main/java/com/blackducksoftware/integration/hub/model/view/components/MetaView.java]: Can we move LinkView into its own file?

**Review by jamesrichard91 — CHANGES_REQUESTED** [review]
Please see my comments in the MetaView class

File: src/main/java/com/blackducksoftware/integration/hub/model/view/components/MetaView.java (hunk 1 above)
Comment (msenmurugan) [reply src/main/java/com/blackducksoftware/integration/hub/model/view/components/MetaView.java]: Removed Super() from the constructor

File: src/main/java/com/blackducksoftware/integration/hub/model/view/components/MetaView.java (hunk 2 above)
Comment (msenmurugan) [reply src/main/java/com/blackducksoftware/integration/hub/model/view/components/MetaView.java]: Created a new LinkView under Components

**Review by jamesrichard91 — APPROVED** [review]
Looks good to me

**Review by psantos1113 — APPROVED** [review]

**Review by psantos1113 — CHANGES_REQUESTED** [review]
Don't use get methods use public fields. Missed that earlier.  ekerwin found it so until that is address I can't approve this.

**Comment by ekerwin** [comment]
Please take a look at the PR for this branch, I think it addresses the final stuff for these changes.

**Review by ekerwin — APPROVED** [review]

File: src/main/java/com/blackducksoftware/integration/hub/model/view/VulnerabilityWithRemediationView.java [hunk 3]
```diff
@@ -29,25 +29,19 @@
...
-
     public double exploitabilitySubscore;
-
     public String source;
-
     public String severity;
-
     public String remediationStatus;
-
+    public String cweId;
```
Comment (stevebillings) [inline src/main/java/com/blackducksoftware/integration/hub/model/view/VulnerabilityWithRemediationView.java:41]: What is cwe? should that maybe be cve?

Comment (msenmurugan) [reply src/main/java/com/blackducksoftware/integration/hub/model/view/VulnerabilityWithRemediationView.java:41]: CWE - Common Weakness Enumeration (trademark of MITRE Corporation). BlackDuck is adding those ids in all vulnerabilities

**Comment by stevebillings** [comment]
Looks good!

**Review by stevebillings — APPROVED** [review]

**Review by psantos1113 — APPROVED** [review]
Looks good

**Pull request merged**